util/train_test_functions.R

```diff
@@ -242,267 +242,142 @@ RestructureTrainedList <- function(train.list){
   
 }
 
-PredictArrayDataWrapper <- function(norm.array.list, train.list, sample.df){
-  # This function takes a list of normalized array data (test set), a 
-  # restructured train.list (from RestructuredTrainedList), and the data.frame
-  # that maps sample names to subtype labels and train/test set labels. It
-  # returns the Kappa statistic from the the various models
+PredictWrapper <- function(train.model.list, pred.list, sample.df, 
+                           return.kap = TRUE, run.parallel = TRUE) {
+  # This function is a wrapper for performing subtype prediction on 
+  # normalized expression data.tables (training data, holdout data, or
+  # reconstructed data) using the models trained on training data
+  # in the supervised analysis (train.list from 2-train_test_brca_subtype.R).
   # 
   # Args:
-  #   norm.array.list: a list of normalized array data (test set)
-  #   train.list: a list of predictive models, restructured using 
-  #               RestructuredTrainedList
+  #   train.model.list: a list of predictive models 
+  #                     (LASSO, linear SVM, random forest) 
+  #   pred.list: list of normalized expression data.tables, labels for this
+  #              data are going to be predicted
   #   sample.df: the data frame that maps sample name/header to subtype and
   #              train/test set labels 
   #              output of 0-expression_data_overlap_and_split.R
+  #   return.kap: logical; should the entire confusionMatrix (FALSE) or just
+  #               the Kappa statistic associated with the prediction?
```

Comment:
Probably should be "should the entire confusion matrix (FALSE) or just the Kappa statistic with the prediction be returned?"